src/Features/Core/Portable/MoveStaticMembers/AbstractMoveStaticMembersRefactoringProvider.cs

```diff
@@ -32,6 +32,12 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 return;
             }
 
+            var syntaxFacts = document.GetRequiredLanguageService<ISyntaxFactsService>();
+            if (syntaxFacts.IsGlobalStatement(memberDeclaration))
```

Comment:
consider a tiny comment explaining things here. 

--

note: the fix seems... odd.  consider inverting and being more like `if (!syntaxFacts.IsMemberDeclaration(memberDeclaration)`

(this could also be done in GetSelectedNodeAsync)